pkgs/games/ckan/default.nix

```diff
@@ -1,44 +1,34 @@
-{ stdenv, fetchFromGitHub, makeWrapper, perl, mono, gtk2, curl }:
+{ stdenv, fetchurl, makeWrapper, mono, gtk2, curl }:
 
 stdenv.mkDerivation rec {
   pname = "ckan";
-  version = "1.16.1";
+  version = "1.28.0";
 
-  src = fetchFromGitHub {
-    owner = "KSP-CKAN";
-    repo = "CKAN";
-    rev = "v${version}";
-    sha256 = "0lfvl8w09lakz35szp5grfvhq8xx486f5igvj1m6azsql4n929lg";
+  src = fetchurl {
```

Comment:
This switches from building from source to downloading a precompiled binary. Building from source is highly prefered in nixpkgs. Did you bump into issues building from source?

Reply:
Sorry, should have given a bit more information.
Yes, I tried a lot to build it from source, but for some reason I keep getting this error:
```
MSBuild auto-detection: using msbuild version '' from '/nix/store/vq32vzsz9v7swaf8jlh0sdvkfkfk5042-msbuild-16.3+xamarinxplat.2019.07.26.14.57/bin'.
Error parsing solution file at [...]/src/CKAN/CKAN.sln: Invalid Image
An error occurred when executing task 'Restore-Nuget'.

[...]

Error: One or more errors occurred. (NuGet: Process returned an error (exit code 1).)
	NuGet: Process returned an error (exit code 1).
```

Seems like [this issue on the CKAN repo](https://github.com/KSP-CKAN/CKAN/issues/2842) might be related, but that was solved a few months ago by simply adding msbuild as a dependency, which doesn't work for me.

I've tried mono5, mono6 (which is 6.0.0 in nixos-stable, maybe this is an issue?), dotnetPackages.NUnitConsole and msbuild, as stated [in the build requirements](https://github.com/KSP-CKAN/CKAN/wiki/Build-Environment) and followed [the build guide](https://github.com/KSP-CKAN/CKAN/wiki/Building-CKAN). In the end, I gave up and reverted to simply use the provided .exe, as that is what is being built anyway.